beta-src/src/utils/map/getOrdersMeta.ts

```diff
@@ -71,13 +80,27 @@ export default function getOrdersMeta(data, phase): Props {
           allowedBorderCrossings = moveChoices.filter((choice) => {
             const { Borders } = choice;
             const from = Borders.find(
-              (border) => border.id === orderUnit.terrID,
+              (border) => border.id === orderUnit?.terrID,
```

Comment:
Why are you checking if orderUnit is truthy here? There is already a truthy if statement on line 50.

Reply:
It errored when I didnt do this. I believe it was saying possibly undefined

Reply:
I do agree its weird that it behaves like this as its in a check for it